phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -211,6 +223,40 @@ protected function writeJsonIndex() {
         v("Index written", VERBOSE_FORMAT_RENDERING);
     }
 
+    protected static function generateDeprecatedInfo($filename) {
+        static $info;
```

Comment:
I'm also for avoiding code duplication and being DRY, but it would mean that we fetch these `$deprecated` data for XHTML format as well. Are we okay with that? I'll fix rest of the issues soon.